tests/tooling/test_codeowners.py

```diff
@@ -34,5 +34,83 @@ def test_codeowners_user_exist():
         assert client.get_user(user[1:])
 
 
+directories_to_test = [
+    'examples',
+    'keras_contrib/activations',
+    'keras_contrib/applications',
+    'keras_contrib/callbacks',
+    'keras_contrib/constraints',
+    'keras_contrib/datasets',
+    'keras_contrib/initializers',
+    'keras_contrib/layers',
+    'keras_contrib/losses',
+    'keras_contrib/metrics',
+    'keras_contrib/optimizers',
+    'keras_contrib/preprocessing',
+    'keras_contrib/regularizers',
+    'keras_contrib/wrappers'
+]
+directories_to_test = [path_to_keras_contrib / x for x in directories_to_test]

```

Comment:
Why the use of pathlib instead of classic `os.getcwd` and `os.path.join` ?

Reply:
Less typing?

Reply:
This test only runs on python 3 on travis btw, so it should be ok to use pathlib.